sdk/python/foundation-models/system/finetune/text-to-image/hftransformers-dreambooth-dog-text-to-image.ipynb

```diff
@@ -0,0 +1,704 @@
+    "foundation_model = max(foundation_models, key=lambda x: int(x.version))\n",
+    "print(\n",
+    "    f\"\\n\\nUsing model name: {foundation_model.name}, version: {foundation_model.version}, id: {foundation_model.id} for fine tuning\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4. Prepare the dataset for fine-tuning the model\n",
+    "\n",
+    "We will use the [dog-example](https://huggingface.co/datasets/diffusers/dog-example) dataset. It consists of 5 dog images."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 4.1 Download the Data\n",
+    "\n",
+    "For dreambooth training, we need few images in the folder. We will download the dog-example dataset locally."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "dataset_dir = \"dog-example\"\n",
+    "if os.path.isdir(dataset_dir) == False:\n",
+    "    !git clone https://huggingface.co/datasets/diffusers/dog-example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from IPython.display import Image\n",
+    "\n",
+    "files = os.listdir(dataset_dir)\n",
+    "image_file = [file for file in files if file.endswith((\".jpg\", \".jpeg\", \".png\"))][0]\n",
+    "sample_image = os.path.join(dataset_dir, image_file)\n",
+    "Image(filename=sample_image, width=400, height=400)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 4.2 Upload the images to Datastore through an AML Data asset (URI Folder)\n",
+    "\n",
+    "In order to use the data for training in Azure ML, we upload it to our default Azure Blob Storage of our  Azure ML Workspace."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Uploading image files by creating a 'data asset URI FOLDER':\n",
+    "\n",
+    "from azure.ai.ml.entities import Data\n",
+    "from azure.ai.ml.constants import AssetTypes\n",
+    "\n",
+    "instance_data = Data(\n",
+    "    path=dataset_dir,\n",
+    "    type=AssetTypes.URI_FOLDER,\n",
+    "    description=\"Dog images for text to image dreambooth training\",\n",
+    "    name=\"dog-images-text-to-image\",\n",
+    ")\n",
+    "\n",
+    "instance_data_uri_folder = workspace_ml_client.data.create_or_update(instance_data)\n",
+    "\n",
+    "print(instance_data_uri_folder)\n",
+    "print(\"\")\n",
+    "print(\"Path to folder in Blob Storage:\")\n",
+    "print(instance_data_uri_folder.path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5. Submit the fine tuning job using `transformers_text_to_image_dreambooth_finetuning_pipeline` component\n",
```

Comment:
```suggestion
    "### 5. Submit the fine tuning job using `diffusers_text_to_image_dreambooth_finetuning_pipeline` component\n",
```

I think better to call it "diffusers" instead of transformers.

Reply:
Changed